Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM  debian:latest
+
+# Credit goes to https://github.com/headstar/sphinx-doc-docker
+
+RUN   apt-get update
+
+RUN   DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip
```

Comment:
I get an error if I try to install python-php textlive... enchant at the same time. `apt-get` fails to fetch resource.. I do not know why, That's why they are separate